src/generate/generate-type-predicates.ts

```diff
@@ -236,6 +237,38 @@ export function generateTypePredicates(
     }`
   }
 
+  const generateJSDocComment = ({
+    type,
+    typeName,
+    isAssertion,
+  }: {
+    type: to.TypeObject
+    typeName: string
+    isAssertion: boolean
+  }): string => {
+    return isAssertion
+      ? `\
+/**
+  * Assert if a variable is of type {@link ${typeName}} and throws a TypeError if the assertion fails.
+  * This function is automatically generated using [type-predicates-generator](https://www.npmjs.com/package/type-predicates-generator).
+  * @param value Argument to inspect.
+  * @throw TypeError if the given argument is not compatible with the type {@link ${typeName}}.
+  */
```

Comment:
It seems to be slightly different from the [JSDoc](https://jsdoc.app/about-getting-started.html) format.
How about the following?

```suggestion
/**
 * Assert if a variable is of type {@link ${typeName}} and throws a TypeError if the assertion fails.
 * This function is automatically generated using [type-predicates-generator](https://www.npmjs.com/package/type-predicates-generator).
 * @param value Argument to inspect.
 * @throw TypeError if the given argument is not compatible with the type {@link ${typeName}}.
 */
```